flow360/component/project.py

```diff
@@ -1289,7 +1300,7 @@ def generate_surface_mesh(
         name: str = "SurfaceMesh",
         run_async: bool = True,
         solver_version: str = None,
-        use_beta_mesher: bool = False,
+        use_beta_mesher: bool = None,
```

Comment:
Does this pass   @pd.validate_call ? Or should we do 
```suggestion
        use_beta_mesher: Optional[bool] = None,
```

Reply:
Ok seems solver version is also None without optional and it worked...